src/usr/lib/rsetup/tui/hardware/hardware.sh

```diff
@@ -261,17 +261,16 @@ Recommendation: fanless or DC fan => power_allocator | PWM fan => step_wise" ||
     fi
 
     selected_governor="$(radiolist_getitem "${RTUI_RADIOLIST_STATE_NEW[0]}")"
-    if enable_unique_config set_thermal_governor "$selected_governor"
+
+    if [[ "$selected_governor" == "power_allocator" ]] && grep -q "pwm-fan" "/sys/class/thermal/cooling_device"*/type
```

Comment:
```suggestion
    if [[ $selected_governor == "power_allocator" ]] && grep -q "pwm-fan" "/sys/class/thermal/cooling_device"*/type <<< ""
```
我们开了nullglob，所以假如在特殊情况没有任何cooling_device或没有type的时候会被bash直接替换成空，这时候会导致grep把程序阻塞住而等待stdin的输入。
这里传一个空字符串进去避免这种情况

另外lint应该会给字符串对比的第一个变量上的引号报错